src/CoreWCF.Http/tests/CoreWCF.Http.SourceGeneration.Tests.csproj

```diff
@@ -0,0 +1,79 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
Would it be worth importing the regular CoreWCF.Http.Tests.csproj file instead of duplicating everything here? I don't want things to get out of sync

Reply:
done